src/styles/blocks/card.css

```diff
@@ -0,0 +1,96 @@
+.card {
+  width: 198px;
+  height: 406px;
```

Comment:
do not set sizes for blocks, if you need it, use mix - create external block for `page` and an element `page__card` to set sizes there